ceph.spec.in

```diff
@@ -1411,7 +1453,34 @@ fi
 
 %files mgr
 %{_bindir}/ceph-mgr
-%{_libdir}/ceph/mgr
+%{_libdir}/ceph/mgr/ansible
+%{_libdir}/ceph/mgr/balancer
+%{_libdir}/ceph/mgr/crash
+%{_libdir}/ceph/mgr/dashboard
+%{_libdir}/ceph/mgr/deepsea
+%{_libdir}/ceph/mgr/devicehealth
+%{_libdir}/ceph/mgr/influx
+%{_libdir}/ceph/mgr/insights
+%{_libdir}/ceph/mgr/iostat
+%{_libdir}/ceph/mgr/localpool
+%{_libdir}/ceph/mgr/mgr_module.*
+%{_libdir}/ceph/mgr/mgr_util.*
+%{_libdir}/ceph/mgr/orchestrator_cli
+%{_libdir}/ceph/mgr/orchestrator.*
+%{_libdir}/ceph/mgr/osd_perf_query
+%{_libdir}/ceph/mgr/pg_autoscaler
+%{_libdir}/ceph/mgr/progress
+%{_libdir}/ceph/mgr/prometheus
+%{_libdir}/ceph/mgr/restful
+%{_libdir}/ceph/mgr/rook
+%{_libdir}/ceph/mgr/selftest
+%{_libdir}/ceph/mgr/smart
+%{_libdir}/ceph/mgr/status
+%{_libdir}/ceph/mgr/telegraf
+%{_libdir}/ceph/mgr/telemetry
+%{_libdir}/ceph/mgr/test_orchestrator
```

Comment:
We don't need to package the test_orchestrator except for the QA.  Relates to #25914

Reply:
since qa installs via packages, we either need t oinclude it here (as we were before) or make a separate ceph-mgr-testing package and then modify teuthology to install that explicitly